shapeshift/web
# feat: make lifi swapper wallet agnostic

## Description

Upgrades lifi swapper to the wallet-agnostic pattern as part of our move to an SDK style integration of swappers in web.

## Pull Request Type


## Issue (if applicable)

progresses https://github.com/shapeshift/web/issues/5024 (but doesn't close it)

## Risk

High risk of broken swappers, especially lifi. Specifically trade execution.

## Testing

* Check all swappers still work
* Check lifi trades still work

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)
![image](https://github.com/shapeshift/web/assets/125113430/4b01e60d-d4a0-456f-a556-32889bf5ec3f)
![image](https://github.com/shapeshift/web/assets/125113430/eee541a4-fef4-4636-85c4-929640e658f1)
